modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -41,9 +41,9 @@ endif::[]
 ifdef::virt-importing-vmware-vm[]
 Importing a VMware VM into {VirtProductName} has the following prerequisites:
 
-* You must have access to an image registry.
-* You must create a VMware Virtual Disk Development Kit (VDDK) image, push it to an image registry, and add it to the `v2v-vmware` ConfigMap.
-* You must power off the VMware virtual machine before you import it.
+* You must create a VMware Virtual Disk Development Kit (VDDK) image, push it to an image registry that is accessible to your {VirtProductName} environment, and add it to the `v2v-vmware` ConfigMap.
+* If you importing a VM with a Windows operating system, you must disable hibernation.
```

Comment:
"If you importing" -> "If you are importing"

Reply:
Fixed